specs/the-example-app-spec.js

```diff
@@ -46,8 +46,8 @@ describe('The Example App', () => {
       cy.get('section.modal .modal__wrapper').should('hidden')
       cy.get('.header__upper-title a').click()
       cy.get('section.modal .modal__wrapper').should('visible')
-      cy.get('section.modal .modal__title').should('contain', 'A reference for developers using Contentful')
-      cy.get('section.modal .modal__content').should('contain', 'This is "The Example App" in node.js. While building your own apps with Contentful,')
+      cy.get('section.modal .modal__title').should('match', '/A reference for .* developers using Contentful/')
+      cy.get('section.modal .modal__content').should('match', '/This is "The .* Example App"\./')
```

Comment:
Can we have it more explicit here? Talked to Mario about this and he actually prefers to have fixed options instead of just allowing everything. I agree with him here, should be more safe for us.

Reply:
You mean like a list of allowed values?

Sure!